# adjusts some configuration defaults to be more reasonable for Debian

diff -Nur licq-1.3.4-RC1/bin/viewurl-lynx.sh licq-1.3.4-RC1.new/bin/viewurl-lynx.sh
--- licq-1.3.4-RC1/bin/viewurl-lynx.sh	2006-08-19 08:05:02.000000000 +0200
+++ licq-1.3.4-RC1.new/bin/viewurl-lynx.sh	2006-09-25 16:04:26.000000000 +0200
@@ -8,7 +8,7 @@
 # Graham Roff, February 1999.
 #
 
-XTERM=xterm
+XTERM=x-terminal-emulator
 LYNX=lynx
 
 ${XTERM} -title "Licq URL View" -e ${LYNX} "$1"
diff -Nur licq-1.3.4-RC1/bin/viewurl-ncftp.sh licq-1.3.4-RC1.new/bin/viewurl-ncftp.sh
--- licq-1.3.4-RC1/bin/viewurl-ncftp.sh	2006-08-19 08:05:02.000000000 +0200
+++ licq-1.3.4-RC1.new/bin/viewurl-ncftp.sh	2006-09-25 16:04:26.000000000 +0200
@@ -9,6 +9,6 @@
 #
 
 NCFTP=ncftp
-XTERM=xterm
+XTERM=x-terminal-emulator
 
 ${XTERM} -title "Licq URL View" -e ${NCFTP} "$1"
diff -Nur licq-1.3.4-RC1/bin/viewurl-w3m.sh licq-1.3.4-RC1.new/bin/viewurl-w3m.sh
--- licq-1.3.4-RC1/bin/viewurl-w3m.sh	2006-08-19 08:05:02.000000000 +0200
+++ licq-1.3.4-RC1.new/bin/viewurl-w3m.sh	2006-09-25 16:04:26.000000000 +0200
@@ -8,7 +8,7 @@
 # Graham Roff, October 1999.
 #
 
-XTERM=xterm
+XTERM=x-terminal-emulator
 W3M=w3m
 
 ${XTERM} -title "Licq URL View" -e ${W3M} "$1"
diff -Nur licq-1.3.4-RC1/plugins/qt-gui/src/optionsdlg.cpp licq-1.3.4-RC1.new/plugins/qt-gui/src/optionsdlg.cpp
--- licq-1.3.4-RC1/plugins/qt-gui/src/optionsdlg.cpp	2006-08-19 08:06:49.000000000 +0200
+++ licq-1.3.4-RC1.new/plugins/qt-gui/src/optionsdlg.cpp	2006-09-25 16:06:37.000000000 +0200
@@ -1503,6 +1503,7 @@
 #ifdef USE_KDE
   cmbUrlViewer->insertItem(DEFAULT_URL_VIEWER);
 #endif
+  cmbUrlViewer->insertItem("sensible-browser");
   cmbUrlViewer->insertItem("viewurl-firefox.sh");
   cmbUrlViewer->insertItem("viewurl-lynx.sh");
   cmbUrlViewer->insertItem("viewurl-mozilla.sh");
diff -Nur licq-1.3.4-RC1/plugins/qt-gui/src/optionsdlg.h licq-1.3.4-RC1.new/plugins/qt-gui/src/optionsdlg.h
--- licq-1.3.4-RC1/plugins/qt-gui/src/optionsdlg.h	2006-08-19 08:06:49.000000000 +0200
+++ licq-1.3.4-RC1.new/plugins/qt-gui/src/optionsdlg.h	2006-09-25 16:05:50.000000000 +0200
@@ -24,7 +24,7 @@
 # define DEFAULT_URL_VIEWER tr("KDE default")
 class KURLRequester;
 #else
-# define DEFAULT_URL_VIEWER "viewurl-mozilla.sh"
+# define DEFAULT_URL_VIEWER "sensible-browser"
 class KURLRequester : public QLineEdit
 {
 public:
diff -Nur licq-1.3.4-RC1/src/icqd.cpp licq-1.3.4-RC1.new/src/icqd.cpp
--- licq-1.3.4-RC1/src/icqd.cpp	2006-08-19 08:04:38.000000000 +0200
+++ licq-1.3.4-RC1.new/src/icqd.cpp	2006-09-25 16:04:26.000000000 +0200
@@ -239,7 +239,7 @@
 
   // Terminal
   m_szTerminal = NULL;
-  licqConf.ReadStr("Terminal", temp, "xterm -T Licq -e ");
+  licqConf.ReadStr("Terminal", temp, "x-terminal-emulator -T Licq -e ");
   m_szTerminal = (char *)malloc(strlen(temp) + 1);
   strcpy(m_szTerminal, temp);
 
diff -Nur licq-1.3.4-RC1/src/licq.conf.h licq-1.3.4-RC1.new/src/licq.conf.h
--- licq-1.3.4-RC1/src/licq.conf.h	2006-08-19 08:04:38.000000000 +0200
+++ licq-1.3.4-RC1.new/src/licq.conf.h	2006-09-25 16:07:18.000000000 +0200
@@ -23,7 +23,7 @@
 "ErrorTypes = 6\n"
 "Rejects = log.rejects\n"
 "Translation = none\n"
-"Terminal = xterm -T Licq -e \n"
+"Terminal = x-terminal-emulator -T Licq -e \n"
 "ProxyEnabled = 0\n"
 "ProxyServerType = 1\n"
 "ProxyServer = \n"
